ui-v2/src/components/ui/run-card/run-card.tsx

```diff
@@ -0,0 +1,151 @@
+import type { components } from "@/api/prefect";
+import {
+	Breadcrumb,
+	BreadcrumbItem,
+	BreadcrumbList,
+	BreadcrumbSeparator,
+} from "@/components/ui/breadcrumb";
+import { Card } from "@/components/ui/card";
+import { Icon } from "@/components/ui/icons";
+import { StateBadge } from "@/components/ui/state-badge";
+import { TagBadgeGroup } from "@/components/ui/tag-badge-group";
+import { Typography } from "@/components/ui/typography";
+import { Link } from "@tanstack/react-router";
+import { cva } from "class-variance-authority";
+import { format, parseISO } from "date-fns";
+import humanizeDuration from "humanize-duration";
+
+const getValues = ({
+	flowRun,
+	taskRun,
+}: {
+	flowRun: undefined | components["schemas"]["FlowRun"];
+	taskRun: undefined | components["schemas"]["TaskRun"];
+}) => {
+	if (taskRun) {
+		const { state, start_time, tags, estimated_run_time } = taskRun;
+		return { state, start_time, tags, estimated_run_time };
+	}
+	if (flowRun) {
+		const { state, start_time, tags, estimated_run_time } = flowRun;
+		return { state, start_time, tags, estimated_run_time };
+	}
+
+	throw new Error("Expecting taskRun or flowRun to be defined");
+};
+
+type Props = {
+	flow?: components["schemas"]["Flow"];
+	flowRun?: components["schemas"]["FlowRun"];
+	/** If task run is included, uses fields from task run over flow run */
+	taskRun?: components["schemas"]["TaskRun"];
```

Comment:
I think we may want to make this `parentRun` and `run` in the future to be more flexible and handle more cases, but this is good for now since it has parity with the current UI.